recipes/obspy/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "1.0.1" %}
+
+package:
+  name: obspy
+  version: {{ version }}
+
+source:
+  fn: obspy-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/o/obspy/obspy-{{ version }}.zip
+  md5: be7ca359c2898038d2da2d8486276bfd
+
+build:
+  number: 0
+  preserve_egg_dir: yes
+  detect_binary_files_with_prefix: true
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - future
+    - numpy
+    - mock  # [py2k]
+    - scipy
+    - matplotlib
+    - lxml
+    - sqlalchemy
+    - argparse # [py26]
+    - flake8
+    - requests
+    - decorator
+  run:
+    - python
+    - setuptools
+    - future
+    - numpy
+    - mock  # [py2k]
+    - scipy
+    - matplotlib
+    - lxml
+    - sqlalchemy
+    - argparse # [py26]
+    - flake8
+    - requests
+    - decorator
+
+test:
+  # Python imports
+  imports:
+    - obspy
+    - obspy.io.mseed
+  commands:
+    - python -c "from obspy import read;"
+
+about:
+  home: http://obspy.org
+  license: LGPLv3
+  summary: "ObsPy: A Python Toolbox for seismology/seismological observatories."
+
+extra:
+  recipe-maintainers:
+    - megies
```

Comment:
@megies just be sure to check with all those guys if they are OK being listed here. They will get a few automatic e-mails once we merge the PR.

@QuLogic nice to see you around here :wink: